### PR TITLE
doc: Update layout.rst, move commands to CODE block

### PR DESCRIPTION
**Pull request by Liuchang0812**
:smile:

### CODE REVIEW

**Comment by dachary** [comment]
jenkins test this please (logs are gone)

**Comment by dachary** [comment]
jenkins test this please (jenkins failure)

**Comment by dachary** [comment]
jenkins test this please

**Comment by alfredodeza** [comment]
Looks like the commits aren't signed. @Liuchang0812 would you mind signing them as per the contributing docs? 

See: https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#signing-contributions

**Comment by Liuchang0812** [comment]
@alfredodeza @liewegas OK, I will do it now. Thanks for your suggestions.

**Comment by Liuchang0812** [comment]
@alfredodeza @liewegas hi, i have tried to sign-off and squash commits.

**Comment by tchaikov** [comment]
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes, in this case, it should be "doc:".

and "update layout.rst" is not helpful to readers, could you be more specific?

File: doc/radosgw/layout.rst [hunk 1]
```diff
@@ -26,19 +26,21 @@ metadata, bucket index, and data.
...
+    $ radosgw-admin metadata list
+    $ radosgw-admin metadata list bucket
+    $ radosgw-admin metadata list bucket.instance
+    $ radosgw-admin metadata list user
+
+    $ radosgw-admin metadata get bucket:<bucket>
+    $ radosgw-admin metadata get bucket.instance:<bucket>:<bucket_id>
+    $ radosgw-admin metadata get user:<user>   # get or set
+    
+    # comments:
```
Comment (Liuchang0812) [inline doc/radosgw/layout.rst]: I have tried this, but it couldn't be sub-list and mixs with `metadata/bucket index/data`. how about  enumerated list. like

1. user: Holds user information
2. bucket: Holds a mapping between bucket name

**Comment by Liuchang0812** [comment]
@tchaikov sorry, I will read the documents, re-send this PR

File: doc/radosgw/layout.rst [hunk 2]
```diff
@@ -21,32 +21,37 @@ Although RADOS only knows about pools and objects with their xattrs and
 omap[1], conceptually RGW organizes its data into three different kinds:
 metadata, bucket index, and data.
 
-* Metadata
+Metadata
+^^^^^^^^^^^^
```
Comment (tchaikov) [inline doc/radosgw/layout.rst]: might want to remove the trailing "^^^^"

File: doc/radosgw/layout.rst [hunk 3]
```diff
@@ -21,32 +21,37 @@ Although RADOS only knows about pools and objects with their xattrs and
...
 
-* Metadata
+Metadata
+^^^^^^^^
 
 We have 3 'sections' of metadata: 'user', 'bucket', and 'bucket.instance'.
 You can use the following commands to introspect metadata entries:
 
-$ radosgw-admin metadata list
+.. code:: bash
```
Comment (alfredodeza) [inline doc/radosgw/layout.rst]: this is not BASH though. Simply by indenting after `::` will format this correctly

**Comment by Liuchang0812** [comment]
@alfredodeza @tchaikov Done

File: doc/radosgw/layout.rst [hunk 4]
```diff
@@ -21,32 +21,37 @@ Although RADOS only knows about pools and objects with their xattrs and
...
 
-* Metadata
+Metadata
+^^^^^^^^
 
 We have 3 'sections' of metadata: 'user', 'bucket', and 'bucket.instance'.
 You can use the following commands to introspect metadata entries:
 
-$ radosgw-admin metadata list
+.. code::
```
Comment (tchaikov) [inline doc/radosgw/layout.rst]: no need to use `.. code::`. simply put `::` at end of the previous line would suffice.

and please remove "* remove the tailing "^^^^^" in radow/layout.rst" from your commit message.

Comment (Liuchang0812) [inline doc/radosgw/layout.rst]: OK

Comment (Liuchang0812) [inline doc/radosgw/layout.rst]: Done

File: doc/radosgw/layout.rst [hunk 5]
```diff
@@ -21,32 +21,35 @@ Although RADOS only knows about pools and objects with their xattrs and
 omap[1], conceptually RGW organizes its data into three different kinds:
 metadata, bucket index, and data.
 
-* Metadata
+Metadata
+^^^^^^^^
 
 We have 3 'sections' of metadata: 'user', 'bucket', and 'bucket.instance'.
-You can use the following commands to introspect metadata entries:
+You can use the following commands to introspect metadata entries:::
```
Comment (tchaikov) [inline doc/radosgw/layout.rst]: the rendered result looks like

You can use the following commands to introspect metadata entries::

could you please just put
```
You can use the following commands to introspect metadata entries::
```

Comment (Liuchang0812) [inline doc/radosgw/layout.rst]: hi, how about:
```
You can use the following commands to introspect metadata entries: ::
```

**Comment by dachary** [comment]
jenkins test this please (unittest_bit_alloc failed)

**Comment by alfredodeza** [comment]
@liewegas seems like all is good here

**Comment by Liuchang0812** [comment]
@liewegas PTAL

**Pull request merged**